### ISSUE TITLE
Definition scope of submodules in YANG 1.1

### ISSUE THREAD

**Issue by svenefftinge** [mbj4668/pyang#323]
RFC 7950 Section 7.2.2 states:

> The mandatory "prefix" substatement assigns a prefix for the module
> to which the submodule belongs. All definitions in the module that
> the submodule belongs to and all its submodules can be accessed by
> using the prefix.

which differs from what is stated in RFC 6020.
Pyang doesn't let me use groupings defined in the module from within a submodule, as it complains with:

`foosub.yang:6: error: grouping "mygrouping" not found in module foosub`

```yang
module foo {
   yang-version 1.1;
   namespace "foo:bar";
   prefix x;
   include foosub;

   grouping mygrouping {
       container c12 {
       }
   }    
}
```
and
```yang
submodule foosub {
   yang-version 1.1;
   belongs-to foo {
       prefix "fooprefix";
   }
   uses "fooprefix:mygrouping" {
   }
}
```

**Comment by mbj4668** [comment]
Yes, this is a known problem.  As a workaround, use YANG 1 style include even in 1.1 modules.

**Comment by mbj4668** [comment]
See also #194

**Comment by anukulverma** [comment]
Fix for this issue is available now ?
